.github/workflows/release_prep_hatch.yml

```diff
@@ -53,6 +53,9 @@ on:
         type: string
         required: false
         default: ''
+      run-unit-tests:
```

Comment:
Allow for optional unit tests; `dbt-tests-adapter` has no tests. This should also probably be updated to something like `unit-test-command` (which could be empty), to follow suit with `build-command` and `check-command`.